### PR TITLE
Add 1.16 release post

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Fix https://github.com/kubewarden/kubewarden.io/issues/254

### CODE REVIEW

**Review by jvanz — CHANGES_REQUESTED** [review]
I think we should mention some policies updates. I just noticed that I've forget to update the card with the necessary information. Let me share the most important ones here:

- https://github.com/kubewarden/container-resources-policy/pull/53
- https://github.com/kubewarden/user-group-psp-policy/pull/96

Sorry for sharing this late
-

**Comment by viccuad** [comment]
@jvanz good point, I totally forgot, thanks! Added them, please re-review.

**Review by jhkrug — APPROVED** [review]
lgtm

File: content/blog/2024/08/kubewarden-1-16-release.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+with a limit that is less than the requested, but will reject the resource
+instead, to force the user to change the minimum request resource amount or
+adjust the policy configuration.
+
+### User-group-psp policy
+
+Starting with `v0.6.2`, if the container image validation is enabled (with
+`validate_container_image_configuration`), the policy will only check for it
+when the rule is "MustRunAs" or "MayRunAs"; "RunAsAny" does not check the
+container image.
```
Comment (jvanz) [inline content/blog/2024/08/kubewarden-1-16-release.md:73]: Actually, the container image is not enabled by default. In the `v0.6.2` we fix an issue in the `RunAsAny` rule for the group when the container image validation is enabled. Furthermore, we documented the issue about the timeout due the time that context aware call takes.

**Review by jvanz — CHANGES_REQUESTED** [review]

**Review by jvanz — APPROVED** [review]

**Pull request merged**